src/mixins/amalgamation-mixin.ts

```diff
@@ -107,9 +107,14 @@ export default class AmalgamationMixin extends Vue {
     return null
   }
 
-  /** Disallow if foreign into ULC if there is also a limited company. */
+  /** Disallow if BC and there is also a foreign, into ULC. */
```

Comment:
'A BC Company cannot amalgamate with an existing foreign corporation to form a BC Unlimited Liability Company.'